src/components/Button/Button.module.css

```diff
@@ -10,15 +10,10 @@
  * 3) Margin 0 added becuase Safari has margin on the button
  */
 .button {
-  @mixin eds-theme-typography-button;
   display: inline-flex;
   align-items: center;
   justify-content: center;
-  padding-top: var(--eds-size-1-and-half);
-  padding-right: var(--eds-size-5);
-  padding-bottom: var(--eds-size-1-and-half);
-  padding-left: var(--eds-size-5);
```

Comment:
Same as above – the sizes now handle their own padding rather than having a default set and overriding it in the sizes.